NOTICES.md

```diff
@@ -0,0 +1,80 @@
+# Implemented notices
+
+Notices are declared in the [`Notice` module](https://github.com/MobilityData/gtfs-validator/tree/master/domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/notice).  Below are details of currently implemented notices.
+
+### Table of Errors
+
+| Error ID      | Error Title         |
+|---------------|---------------------------|
+| [E001](#E001) | | 
+| [E002](#E002) | | 
+| [E003](#E003) | | 
+| [E004](#E004) | | 
+| [E006](#E006) | | 
+| [E009](#E009) | | 
+| [E010](#E010) | | 
+| [E011](#E011) | | 
+| [E012](#E012) | |
+| [E013](#E013) | |
+| [E015](#E015) | |
+| [E016](#E016) | | 
+| [E017](#E017) | |
+| [E018](#E018) | | 
+| [E019](#E019) | | 
+| [E020](#E020) | | 
+| [E021](#E021) | | 
+| [E022](#E022) | Invalid Language Code | 
+| [E023](#E023) | Invalid Email | 
+
+### Table of Warnings
+
+| Warning ID    | Warning Title             |
+|---------------|---------------------------|
+| [W001](#W001) | | 
+| [W002](#W002) | | 
+| [W003](#W003) | | 
+| [W004](#W004) | | 
+| [W005](#W005) | Route description equals Route name | 
+| [W006](#W006) | Route Color Contrast | 
+
+# Errors
+
+<a name="E022"/>
+
+### E022 - Invalid Language Code
+
+Language codes used in a GTFS feed should be under the IETF BCP 47 format. Please visit links below for an introduction to IETF BCP 47.
+
+#### References:
+* [Field Types Description](http://gtfs.org/reference/static/#field-types)
+* [IETF BCP 47 Language Tags Introduction](https://www.w3.org/International/articles/language-tags/)
+
+<a name="E023"/>
+
+### E023 - Invalid Email
+
+An email should be a valid email address.
+
+#### References:
+* [Field Types Description](http://gtfs.org/reference/static/#field-types)
+
+# Warnings
+
+<a name="W005"/>
+
+### W005 - Route description equals Route name
+
+A Route description should provide useful information about the Route, and should not duplicate from the Route long or short name. Although, the Route description may reuse the Route names to give explanations.
```

Comment:
A few comments:
* I'd be very careful with the language describing these types of rules, as it will be easy to accidentally imply something that goes against the spec or best practices. Whenever possible please link to and quote the spec or best practices.
* Please reference field names using `code_formatting`.
* Given the below quoted text from the spec, this should be an error, not a warning (the spec says "Do not simply duplicate the name of the route")

An example here would be:

---

The GTFS spec defines `routes.txt` [`route_description`](https://gtfs.org/reference/static/#routestxt) as:

>Description of a route that provides useful, quality information. Do not simply duplicate the name of the route.

See the GTFS and GTFS Best Practices links below for more examples of how to populate the `route_short_name`, `route_long_name`, and `route_description` fields.

#### References:
* [GTFS `routes.txt`](http://gtfs.org/reference/static/#routestxt)
* [GTFS `routes.txt` Best Practices](https://gtfs.org/best-practices/#routestxt)

Reply:
I'll change this the text.

Also, for now, I created warning notices this one and for route color constrast because the fields were optional (Route description, color and text color). Should it be errors instead of warnings?

Reply:
After discussion with Fabrice, we changed these to be errors instead.

Reply:
Agree - even if a field is optional, if the rule is stated in the GTFS spec it should be an error. But I don't think it says anything about color contrast in the GTFS spec itself? If not, then it should be a warning.